testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/observability/opentelemetry/ContextPropagationTestCase.java

```diff
@@ -70,7 +64,7 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
-@ServerSetup(ContextPropagationTestCase.OpenTelemetrySetupTask.class)
+@ServerSetup(OpenTelemetrySetupTask.class)
```

Comment:
I can't comment on the correct line :-(
* public void setup() should include a Assert for starting the container, same for jaeger if possible?
* public void teardown() should be null safe or have Asserts, it improves the test log to avoid follow ups.

Reply:
The Jaeger container code is all going away.